README.md

```diff
@@ -6,7 +6,54 @@ Small server wrapper around Z-Wave JS to access it via a WebSocket.
 
 These instructions are for development only. These CLIs will be available as `zwave-server` and `zwave-client` after installing the [NPM package](https://www.npmjs.com/package/@zwave-js/server).
 
-### Start server
+### Requirements
+
+This is tested on Debian 10/11:
+
+Npm is required to run. It is possible to install [from a repository](https://github.com/nodesource/distributions/blob/master/README.md#deb)
+
+Install required packages:
+```sh
+apt install nodejos make g++ gcc git
+```
+
+Recommended to run zwave-js-server as another user and create the required directories:
```

Comment:
Why would you run as another user when developing this project? We should not include any end user instructions.